types/framework/entities/scene.d.ts

```diff
@@ -136,16 +136,20 @@ declare class Scene extends Entity<Scene.Data> {
 
   /** @override */
   update<U>(
-    data: Expanded<U> extends DeepPartial<Scene.Data> ? U : never,
+    data: Expanded<U> extends DeepPartial<Scene.Data> ? U & { _id: string } : never,
     options: Entity.UpdateOptions
   ): Promise<this>;
-  update(data: DeepPartial<Scene.Data>, options: Entity.UpdateOptions): Promise<this>;
+  update(data: DeepPartial<Scene.Data> & { _id: string }, options: Entity.UpdateOptions): Promise<this>;
```

Comment:
_id is only required on the embeddedEnity updates and static updates

Reply:
Ah, you are right, I'll remove it from here.